components/conversation/src/Conversation.svelte

```diff
@@ -21,6 +25,7 @@
     Conversation,
     Account,
   } from "@commons/types/Nylas";
+  import "../../contacts-search/src/ContactsSearch.svelte";
```

Comment:
Is this how we want to nest components? I thought we'd import from npm as a dev dependency?